### ISSUE TITLE
Support papersize variable when converting .md to .docx

### ISSUE THREAD

**Issue by ad-si** [jgm/pandoc#6550]
```sh
$ pandoc --version
pandoc 2.9.2.1
Compiled with pandoc-types 1.20, texmath 0.12.0.1, skylighting 0.8.3.2
Default user data directory: /Users/adrian/.local/share/pandoc or /Users/adrian/.pandoc
Copyright (C) 2006-2020 John MacFarlane
Web:  https://pandoc.org
This is free software; see the source for copying conditions.
There is no warranty, not even for merchantability or fitness
for a particular purpose.

$ cat test.md
---
title: Test
papersize: a4
---

This is just a test!

$ pandoc --output test.docx test.md
```

The test.docx now has a size of 215.9 mm x 279.4 mm (US Letter) and not A4.

**Comment by agusmba** [comment]
AFAIK, `papersize` is only supported on latex-like outputs.
If you want to change the default page size in docx, I would suggest using a custom reference doc.

Whether pandoc's internal default reference docx has sensible-sized pages is something I haven't looked at, since I always use my own reference doc for docx output .

**Comment by agusmba** [comment]
BTW, I just repeated your test (with pandoc version 2.9.1.1 and 2.9.2.1 running on windows and using the official docker image) and both tests have standard "letter" size 215.9mm x 279.4mm, so there must be something else strange on your side.

**Comment by ad-si** [comment]
Well, then this issue is a feature request that `papersize` should be supported for .docx files 😉.

Mh, seems to be a problem of Apple Pages. In LibreOffice it also has the standard letter size.

**Comment by agusmba** [comment]
Fair enough (you might want to change the title of this feature request then).
In the meantime, I'd still suggest that you use a reference-doc with your desired dimensions. This will allow you to also tweak the styling to your hearts content.
BR.

**Comment by ad-si** [comment]
> In the meantime, I'd still suggest that you use a reference-doc with your desired dimensions. This will allow you to also tweak the styling to your hearts content.

If I just want to convert to docx once in a while, that's a little overkill.

**Comment by ant-222** [comment]
*d-si*

> If I just want to convert to docx once in a while, that's a little overkill.

I think it is all right to specify all *docx* settings in the reference document, especially because there are so many. Deciding for which settings to provide alternative means of specification is a fuzzy matter. Once `papersize' is supported, someone else will ask about margins, fonts, *&c*...

**Comment by ad-si** [comment]
> Once papersize is supported, someone else will ask about margins, fonts, &c...

Sure, the decision which to include and which not to include must be done by someone. But to simplify nothing, because you can't simplify everything sounds like questionable reason. Basic settings with advanced settings in another UI is a standard and well established UX pattern.

**Comment by ant-222** [comment]
*ad-si*, but I do not consider your proposal a simplification, because making the necessary adjustments in the reference document is not difficult to begin with, and has the following advantages:

1. all settings are specified and stored in one place,
2. settings are kept separate from the document source, allowing you to process the same document with different settings without modifying the document.

Such is my opinion, but I am just a random user of *Pandoc* who has dropped in to see what other users are requesting, so I won't argue any more about it.

**Comment by bmuyambo-dot** [comment]
Here’s a DOCX-friendly alternative format in Markdown. You can convert this to Word easily with Pandoc (or copy-paste into Word with minimal reformatting). The structure mirrors an NSC Paper II style (Section A and Section B), with 12 Section A questions and 6 Section B questions, keeping concepts and difficulty balanced. ASCII diagrams are embedded as code blocks for easy viewing in plain text.

Copy this into a file named paper.md, then convert to DOCX with Pandoc:
- Command (recommended): pandoc paper.md -s -o paper.docx
- If you don’t have Pandoc, open the Markdown in a Markdown editor and export to DOCX, or paste into Word and apply styles manually.

NSC Mathematics: Paper II (IEB-style) — Markdown Version
Time: 3 hours
Total marks: 150

Section A (75 marks)
Answer all questions in Section A. Each question contains subparts with allocated marks shown in brackets.

QUESTION 1 – Functions and graphs (6 marks)
(a) Determine the domain of f(x) = sqrt(4 − 2x). (2)
(b) For f(x) = (3x − 5)/(x + 2), find:
   - x-intercept(s) and y-intercept; (2)
   - the domain of f. (2)

ASCII sketch (simple rational function graph):
```
      y
      ^
      |       /
      |     /  .
      |---/-------> x
      |  /  .
      | /
      -----------------
```

QUESTION 2 – Sequences (6 marks)
(a) An arithmetic sequence has T1 = 5 and T3 = 17. Find T2 and the general term Tn. (3)
(b) A geometric sequence has T1 = 4 and r = 1/2. Find T4 and S4 (the sum of the first four terms). (3)

QUESTION 3 – Circle geometry and trigonometry (6 marks)
(a) In a circle of radius 6 cm, a chord AB subtends a central angle of 60 degrees. Find the length AB. (3)
(b) For a right triangle with angles 30°, 60°, 90° and hypotenuse 8, determine the lengths of the two legs. (3)

QUESTION 4 – Analytic geometry: lines and distance (6 marks)
(a) Find the equation of the line passing through A(2, −1) and B(−3, 4). (2)
(b) Compute the perpendicular distance from the origin to this line. (2)
(c) Find the intersection point of the lines 2x + y = 4 and x − y = 1. (2)

QUESTION 5 – Calculus: derivatives (6 marks)
(a) Differentiate f(x) = 2x^3 − 3x^2 + x. (2)
(b) Find the equation of the tangent to the curve y = f(x) at x = 1. (2)
(c) Determine the intervals on which f is increasing or decreasing. (2)

QUESTION 6 – Statistics and data representation (6 marks)
(a) Given the data set: 5, 7, 7, 9, 12, compute the mean and median. (3)
(b) Represent the data 4, 7, 7, 8, 8, 10, 12, 12 in a simple histogram-like ASCII diagram. (3)

QUESTION 7 – Probability basics (6 marks)
(a) A fair six-sided die is rolled. What is P(even)? (2)
(b) Two coins are flipped. What is P(at least one head)? (2)
(c) A bag contains 3 red, 2 blue and 5 green marbles. If one marble is drawn, what is P(not blue)? (2)

QUESTION 8 – Financial mathematics: simple and compound interest (6 marks)
(a) You invest R 1 000 at a simple interest rate of 5% per year for 3 years. What is the amount after 3 years? (3)
(b) The same principal invested at 5% per year compounded annually for 3 years yields what amount? (3)

QUESTION 9 – Algebra: quadratic equations and inequalities (6 marks)
(a) Solve x^2 − 3x − 4 = 0. (2)
(b) Solve the inequality x^2 − 2x − 3 ≤ 0. (2)
(c) Factor and solve 4x^2 − x − 3 = 0. (2)

QUESTION 10 – Vectors in 2D (6 marks)
(a) Given A(1, 2) and B(−2, 3), compute the vector AB and its magnitude. (2)
(b) Let C(2, 0). Compute the dot product AB · AC and use it to determine the angle between AB and AC (use cos θ = (AB · AC)/(|AB||AC|)). (2)
(c) Determine whether AB is perpendicular to BC where B(−2, 3) and C(2, 0). Show your work. (2)

QUESTION 11 – Transformations and graphs (6 marks)
(a) Let f(x) = x^2. Write the equation of the transformed graph after translating 3 units to the right and 2 units up. (2)
(b) Describe the effect on the graph of y = f(x) if we reflect it in the x-axis. (1)
(c) If g(x) = f(2x), explain how the graph of f is affected (horizontal compression by a factor of 2). (3)

QUESTION 12 – Definite integrals and area (6 marks)
(a) Evaluate ∫ from 0 to 3 (2x + 1) dx. (3)
(b) Find the area between y = x^2 and y = 2x on the interval [0, 2]. (3)

Section B (75 marks)
Answer all questions in Section B. Each question is longer and multi-part, designed to integrate concepts and require more extensive reasoning. ASCII diagrams are embedded where helpful.

QUESTION 13 – Modelling, probability and combinatorics (13 marks)
A small bakery records daily sales (in units) over a week as follows: Mon 14, Tue 18, Wed 15, Thu 20, Fri 16, Sat 22, Sun 19.

(a) Compute the mean daily sales for the week. (2)
(b) Construct a simple boxplot representation (ASCII) of the weekly data. Include minimum, Q1, median, Q3, and maximum. (4)
(c) If a random day is chosen, what is the probability that sales exceed 18 units? (2)
(d) Suppose you want to form a 3-day sales forecast using a simple average of the last three days’ sales (Thu–Sat). What is the forecast for Sunday? (2)
(e) If the daily sales follow a roughly normal distribution with the same mean and a standard deviation of 2.8 units, estimate the probability that a randomly chosen day has sales between 15 and 20 units. (3)

ASCII boxplot example:
```
Min ─────────┬────────┬──────── Median ────────┬──────── Max
      14       15       18           20
```

QUESTION 14 – Circle geometry and trigonometric applications (12 marks)
A circle with radius 8 cm has a chord CD of length 12 cm. Let O be the center, and the chord subtends angle ∠COD at the center.

(a) Find ∠COD in degrees. (3)
(b) Find the distance from the center O to the chord CD. (3)
(c) A tangent is drawn at point D. If the radius OD is perpendicular to the tangent, find the length of the tangent segment from D to the point of tangency T on a line extended from CD. (3)
(d) If an arc measure corresponding to ∠COD is converted to radians, what is the length of the arc CD? (3)

QUESTION 15 – Vectors and forces (12 marks)
A physics scenario: Two forces act on an object at a point P. F1 = (4, −1) N, F2 = (−2, 3) N.

(a) Compute the resultant force vector F = F1 + F2 and its magnitude. (4)
(b) Determine the angle that F makes with the positive x-axis. (4)
(c) If the object is subjected to a third force F3 = k(−3, −1) N which makes the magnitude of F3 minimal while keeping the resultant force magnitude equal to 5 N, find k (solve for k). (4)

QUESTION 16 – Calculus: optimization and area under a curve (12 marks)
A company’s revenue is modeled by R(x) = −x^3 + 6x^2 + 8x, where x is the number of units sold (in thousands). Cost C(x) = x^2 + 4x + 3 (in thousands).

(a) Profit P(x) = R(x) − C(x). Find P′(x) and determine the critical points. (4)
(b) Determine the production level x that maximizes profit (local maximum). (4)
(c) Compute the maximum profit value at that x. (4)

QUESTION 17 – Financial mathematics: annuities and loans (13 marks)
A borrower considers two options for repaying a loan of R 500 000 over 20 years at an annual interest rate of 7% (compounded monthly).

Option A: level monthly repayments over 240 months.
Option B: a payment holiday of 12 months (no payments for the first year), followed by level payments for the remaining 228 months at a slightly adjusted rate to reflect the postponed payments.

(a) Compute the monthly repayment for Option A. (6)
(b) Discuss qualitatively how a 12-month holiday would affect the total interest paid in Option B relative to Option A, assuming the same nominal rate and that postponing payments increases the principal and thus interest. (2)
(c) Without doing full calculations, explain one advantage and one disadvantage of postponing payments for the first year. (5)

QUESTION 18 – Integration, area and applications (13 marks)
A regional park is bounded by the curves y = x^2 and y = 4 − x^2 from x = −1 to x = 2, and by the line y = 0 on the x-axis for the lower boundary in parts of the region.

(a) Compute the total area of the region enclosed by these curves. (5)
(b) If the park adds a straight boundary along the line y = x from x = 0 to x = 2, compute the area of the triangular portion cut off by this line within the region, and express the remaining area. (4)
(c) Interpret briefly how definite integrals reflect the accumulation of area in this context. (3)
(d) Sketch, in ASCII, the rough shape of the two curves y = x^2 and y = 4 − x^2 between x = −1 and x = 2, and the line y = x to visualize the described region. (1)

Notes and guidance
- The questions in Section A test core NSC Mathematics competencies: functions and graphs, sequences, circle and plane geometry, analytic geometry, calculus basics, statistics and data, probability, and finance. Each question includes subparts to scaffold difficulty and enable partial credit.
- Section B includes longer, integrative problems combining ideas (calculus with optimization, finance with time value of money, vectors with physics, and statistics with data interpretation).
- ASCII diagrams are embedded where helpful. If you’d like, I can provide a clean, formatted Word-ready version or a LaTeX source.

Conversion tips
- To produce a Word document from this Markdown:
  - Use Pandoc: pandoc paper.md -s -o paper.docx
  - Or paste the content into a Markdown editor and export to DOCX, then apply Word styles (Title, Heading 1/2, etc.).
- If you’d prefer a LaTeX version or an HTML version, I can provide those as well.

Would you like me to attach a clean LaTeX version or a pre-rendered HTML file in addition to this Markdown?

**Comment by rnwst** [comment]
I agree with @ant-222 that the right place to handle this is the reference-doc, and that can already be done. Creating a reference-doc is very simple, and is not enough of a hurdle to justify adding yet another variable or option. Furthermore, if possible all settings should be defined in one place, and that place is the reference-doc. I do, however, agree with @ad-si that the default page size should be A4, not US letter size. I'll submit a PR for that.
